### DISCUSSION TITLE
inner anything

### DISCUSSION THREAD

**Discussion by comod** [JetBrains/ideavim#444] [Q&A]
Hi,
is it possible to create a motion like `i"` ect combined with others with a logical `OR` to a single "shortcut"-version of `i`?
eg: `vy` = (y = an`y`thing just for example) would do something similar to `vi`,`"` or `p` or `(`

**Comment by AlexPl292** [comment]
Hmm, I personally can't remember any plugin of functionality like that. @lippfi any ideas?
